worker/include/Settings.hpp

```diff
@@ -38,7 +38,9 @@ class Settings
 		uint16_t rtcMaxPort{ 59999u };
 		std::string dtlsCertificateFile;
 		std::string dtlsPrivateKeyFile;
-		std::string libwebrtcFieldTrials{ "WebRTC-Bwe-AlrLimitedBackoff/Enabled/" };
+		std::string libwebrtcFieldTrials{
+			"WebRTC-Bwe-AlrLimitedBackoff/Enabled/WebRTC-Bwe-LossBasedBweV2/Enabled:true,CandidateFactors:1.02|1.0|0.95,DelayBasedCandidate:true,HigherBwBiasFactor:0.0002,HigherLogBwBiasFactor:0.02,ObservationDurationLowerBound:250ms,InstantUpperBoundBwBalance:75kbps,BwRampupUpperBoundFactor:1000000.0,InstantUpperBoundTemporalWeightFactor:0.9,TemporalWeightFactor:0.9,MaxIncreaseFactor:1.3,NewtonStepSize:0.75,InherentLossUpperBoundBwBalance:75kbps,LossThresholdOfHighBandwidthPreference:0.15,NotIncreaseIfInherentLossLessThanAverageLoss:true,TrendlineIntegrationEnabled:true,SendingRateSmoothingFactor:0.6/"
```

Comment:
This string has more importance than at this moment it seems. Is it possible to separate the key,values with a newline instead of a `|`? Though its beauty, but its really hard to see what is the config default from the code at this point.

Reply:
We will need to modify libwebrtc parser for this, but i will take a look.

Reply:
Might not need to change the libwebrtc parse, though it is ported, might just make a function replace every newline character to a |. But the this is stupidly minor, if a documentation surfaces these metrics with a default value that's the best! and then we don't need to change here at all.